tests/inspect/inspect.go

```diff
@@ -213,8 +251,11 @@ func getOwnCgroupPath(controller string) (string, error) {
 		if len(parts) < 3 {
 			return "", fmt.Errorf("error parsing /proc/self/cgroup")
 		}
-		if parts[1] == controller {
-			return parts[2], nil
+		controllerParts := strings.Split(parts[1], ",")
+		for _, c := range controllerParts {
+			if c == controller {
+				return parts[2], nil
+			}
```

Comment:
The change LGTM.

About the TODO in this function: what about moving the cgroup function in `pkg/cgroup/` and then just import 
"github.com/coreos/rkt/pkg/cgroup"?